deepwell/src/services/view/service.rs

```diff
@@ -100,12 +100,12 @@ impl ViewService {
         let options = PageOptions::parse(page_extra);
 
         // Get page, revision, and text fields
-        let (category_slug, page_slug) = split_category(page_slug);
+        let (category_slug, page_clean_slug) = split_category(page_slug);
         let page_info = PageInfo {
-            page: cow!(page_slug),
+            page: cow!(page_clean_slug),
             category: cow_opt!(category_slug),
             site: cow!(&site.slug),
-            title: cow!(page_slug),
+            title: cow!(page_clean_slug),
```

Comment:
"Clean" because the slug is normalized? Is this the cause of the improper normalization issue you reported in chat?

Reply:
It was because deepwell receives the per category slug and not the full slug, and couldn't resolve pages with category properly, so I had to separate the variables for them to work. The variable name was intended to mean something like "stripped down slug".